support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx

```diff
@@ -109,6 +109,9 @@ const createCssModifiers = () => {
 // JTL - TBD - Delete after Moment is done running
 const cssModifiers = createCssModifiers();
 
+const showSecureTransactionIndicator = (store.getState().common.abParticipations.paymentSecurityDesignTest === 'V1_securetop' && countryGroupId !== 'GBPCountries')
```

Comment:
is the `&& countryGroupId !== 'GBPCountries'` part redundant here?

Reply:
Yes, will amend!